bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -1790,17 +1790,9 @@ pimcore.helpers.editmode.openLinkEditPanel = function (data, callback) {
         listeners: {
             keyup: function (el) {
                 const value = el.getValue();
+                const pathRegex = new RegExp('^((\\/)|((\\/[^\\/]+)+(\\/)?))$');
```

Comment:
I don't think the escaping of the slashes is necessary.

```suggestion
                const pathRegex = new RegExp('^((/)|((/[^/]+)+(/)?))$');
```

Reply:
Also, I don't get why you defined so many groups. It should be fine like this:

```suggestion
                const pathRegex = new RegExp('^(/|(/[^/]+)+/?)$');
```

Reply:
Ah yes, that was something PhpStorm did after copy and pasting the pattern. Nicely spotted, thanks!